hazelcast/src/test/java/com/hazelcast/internal/metrics/jmx/JmxPublisherTest.java

```diff
@@ -285,4 +290,17 @@ public void when_shutdown_noMBeansLeak() throws Exception {
     private MetricDescriptor newDescriptor() {
         return DEFAULT_DESCRIPTOR_SUPPLIER.get();
     }
+
+    private BiTuple<String, List<TriTuple<String, Number, Type>>> metric(String objectName,
+                                                                           List<TriTuple<String, Number, Type>> measurements) {
```

Comment:
nit: padding seems wrong here (2 extra spaces)

Reply:
Hm :thinking: Lazy IDE :smile: Thanks, fixed here and in the other PR too.